src/index.ts

```diff
@@ -390,6 +390,10 @@ function getMapSettings(settings: Partial<Settings> | undefined): SavedSettings
         return {};
     }
 }
+/** Validate HTML element */
+function isHTMLElement(o: unknown) {
+    return o instanceof HTMLElement;
+}
```

Comment:
```suggestion
```

since the function is really small, you can move the code inline in the checks themselves